lib/batch.js

```diff
@@ -0,0 +1,49 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const logger = require('./logging')('batch');
+const P = require('./promise');
+
+function inject(server, options) {
+  return new P(function(resolve) {
+    server.inject(options, resolve);
+  });
+}
+
+function batch(request, map) {
```

Comment:
A high-level comment on what this function does would be helpful and the expected structure of its arguments would be helpful.